src/app/participant-list/participant-list.component.ts

```diff
@@ -2100,4 +2133,20 @@ export class ParticipantListComponent implements OnInit {
     }
     return false;
   }
+
+  addAutomatedScoringColumns(): void {
+    this.showComputedObject = true;
+
+    this.dataSources.set("computed", "Morning-Evening Questionnaire Scoring");
```

Comment:
Could we get these columns dynamically? If another study does scoring they would be called differently

Reply:
is that information added from the DSS site into `activity_definition`

Reply:
@SimoneMaiwald it's added as `computed` key in ES only for `participants_structured.circadia.circadia`

Reply:
so we don't have a way of getting a "nice" text/name for the keys under the computed object?

Reply:
@SimoneMaiwald Unfortunately no, because this task is too `Circadia` - specific

Reply:
ok, then lets come up with a plan when there is another study using it